app/views/results/index.html.erb

```diff
@@ -1 +1,8 @@
 <h1 class='m-4 text-center'>Your Results</h1>
+
+<% if current_user.ratings_needed > 0 %>
```

Comment:
Great place for a model method, I like that we can use the same one in both spots too